src/modules.js

```diff
@@ -103,6 +103,7 @@ var LibraryManager = {
     if (AUTO_JS_LIBRARIES) {
       libraries = libraries.concat([
         'library_webgl.js',
+        'library_webgpu.js',
```

Comment:
As there is an explicit option for this, this could be added conditionally on that option (see some other libraries lower down).